contributing_to_docs/doc_guidelines.adoc

```diff
@@ -381,6 +381,11 @@ include::modules/module-filename.adoc[leveloffset=+1] <1>
 <1> `[leveloffset=]` link:https://docs.asciidoctor.org/asciidoc/latest/directives/include-with-leveloffset/#manipulate-heading-levels-with-leveloffset[is relative], and changes the header level of included content.
 
 You can use `tag` attributes to link:https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions[define regions of a file and include by tag instead of the whole file.] This enables you to make conditional modifications to existing modules instead of duplicating content. When creating a tag, use a tag name that makes it easy to understand where the content defined will be included.
++
+[WARNING]
+====
+Some functionalities of tagged regions are not fully supported by PV1. Use with caution and verify that access.redhat.com builds and docs.openshift.com builds are including or excluding content as expected. 
```

Comment:
I think we should add here that some of these unsupported features will pass all checks and fail only late in the Pv1 publication process, so our normal guardrails won't protect us from these.

Should we include known problems in Pv1 that render correctly in the docs.openshift.com build that we have seen in [troubleshooting](https://docs.google.com/document/d/165PuwDWjuUxRFBfG7-QWTQhXjaifBVuraEUiv156Wbs)? 

1. seems like wildcard (`*`) is not supported
2. Pv1 doesn't recognize `tags=` (plural) in the `include` line, only `tag=` (singular)
3. we should escape tags in YAML blocks (not actually sure if this one chokes Pv1, but we should do it even though the docs.openshift.com build is fine if we don't)

Reply:
@jeana-redhat late failure is definitely a good thing to note. 

I agree that we should include some of these issues we've seen in Troubleshooting:
1. I was going to ask in that working thread today if we could safely assume that wildcards are not supported now after your tests the past few business days. Sounds like the answer is yes, and if that's the case, we should def note it. 
2. Interesting....was this with use of a wildcard? Because I use "tags" in my PR that worked: https://files.slack.com/files-pri/T030G10V24F-F074P0LDQMR/screenshot_2024-05-22_at_2.08.55___pm.png
3. Yeah the escape within YAML blocks is a good thing to make extra clear. I think we punted to the asciidoc documentation for this one, but its an easy detail to miss there.

Reply:
re: 2. when I used `tags`, the GitLab preview rendered everything, including the lines in the module defining tagged regions. When I changed it to singular, GitLab finally looked correct. 

Since plural worked for you, I bet there is some other part of the interaction that is different between the type of content I added vs what you did. So maybe a flat statement that it doesn't work is inaccurate, but it does seem to be the case that it doesn't _always_ work. Unless we know why, might be best to avoid using plural :thinking:

Reply:
Update from chat w/ Sara about sing/plur `tag` in the `include` line. We know:

- multiple tags with `tags` is fine
- single tag with `tags` is bad
- single tag with `tag` is fine
- multiple tags with `tag` is unknown (not tested)

Based on this, we are thinking that we can recommend to make the sing/plur agree (i.e., use singular `tag` for one tag, use plural `tags` for multiple) because we know this works in the Pv1 build.

If an opportunity to test multiple tags with `tag` presents itself, we could do that to see if we could simplify the guidance.

Reply:
@jeana-redhat sorry I dropped this between a release date and PTO. I updated these guidelines based on your feedback and I tried to add some examples in there too to hopefully make it a little clearer. When you have time, let me know what you think. I know @libander was also interested in doing some more testing on Wildcard functionality.